src/Orleans/Serialization/SerializationContext.cs

```diff
@@ -7,6 +7,11 @@ namespace Orleans.Serialization
     public interface ICopyContext
     {
         /// <summary>
+        /// Gets the serialization manager.
+        /// </summary>
+        SerializationManager SerializationManager { get; }
```

Comment:
should you also add IServiceProvider to this and ISerializationContext, just to have symmetry with IDeserializionContext? I don't really care, but my OCD found this asymmetry a little odd.